controllers/chat/group/groupAddMembers.js

```diff
@@ -68,21 +69,39 @@ const groupAddMembers = async (req, res) => {
       const textTargetUser = `${ownUser.username} added you to this group`;
       const textDefaultUser = `${ownUser.username} added ${targetUserModel.username} to this group`;
 
-      await channelToAdd.sendMessage({
-        text: textDefaultUser,
-        own_text: textOwnUser,
-        other_text: textTargetUser,
-        other_system_user: member,
-        better_type: 'add_member_to_group',
-        type: 'system',
-        user_id: userId,
-        only_to_user_show: userId,
-        disable_to_user: false,
-        is_from_prepopulated: true,
-        system_user: userId,
-        isSystem: true,
-        members: [member]
-      });
+      await channelToAdd.sendMessage(
+        {
+          text: textDefaultUser,
+          own_text: textOwnUser,
+          other_text: textTargetUser,
+          other_system_user: member,
+          better_type: 'add_member_to_group',
+          type: 'system',
+          user_id: userId,
+          only_to_user_show: userId,
+          disable_to_user: false,
+          is_from_prepopulated: true,
+          system_user: userId,
+          isSystem: true,
+          members: [member]
+        },
+        {
+          skip_push: true
+        }
+      );
+
+      const other_members = all_members.filter((m) => m !== member && m !== userId);
+      try {
+        await BetterSocialCore.fcmToken.sendGroupChatNotification(userId, textOwnUser);
+      } catch (error) {
+        console.error('Failed to send group chat notification to user', error);
+      }
+      await BetterSocialCore.fcmToken.sendGroupChatNotification(member, textTargetUser);
+      await Promise.all(
+        other_members.map(async (m) => {
+          await BetterSocialCore.fcmToken.sendGroupChatNotification(m, textDefaultUser);
+        })
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling for `sendGroupChatNotification` has been improved by adding a try-catch block around the call for the `userId`. However, there is no error handling for the calls to `sendGroupChatNotification` for `member` and `other_members`. It would be better to add error handling for these as well to avoid unhandled promise rejections.
```diff
-      await BetterSocialCore.fcmToken.sendGroupChatNotification(member, textTargetUser);
+      try {
+        await BetterSocialCore.fcmToken.sendGroupChatNotification(member, textTargetUser);
+      } catch (error) {
+        console.error('Failed to send group chat notification to member', error);
+      }

-      await Promise.all(
-        other_members.map(async (m) => {
-          await BetterSocialCore.fcmToken.sendGroupChatNotification(m, textDefaultUser);
-        })
-      );
+      try {
+        await Promise.all(
+          other_members.map(async (m) => {
+            await BetterSocialCore.fcmToken.sendGroupChatNotification(m, textDefaultUser);
+          })
+        );
+      } catch (error) {
+        console.error('Failed to send group chat notification to other members', error);
+      }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->